tests/max_config/linux/nidaqmxMaxConfig.ini

```diff
@@ -188,6 +188,15 @@ BusType = PCIe
 PCI.BusNum = 0x0
 PCI.DevNum = 0x0
 
+[DAQmxDevice digitalPatternTester]
```

Comment:
I'm fairly certain HSDIO was never ported to Linux Desktop. I think we need to remove this.

Reply:
Confirmed via https://www.ni.com/en/support/documentation/compatibility/21/ni-hardware-and-operating-system-compatibility.html